app/api/orders.py

```diff
@@ -319,7 +319,7 @@ def before_update_object(self, order, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        if data.get('amount') or data.get('is_billing_enabled'):
+        if data.get('amount') or data.get('is_billing_enabled') or order.event.is_billing_info_mandatory:
```

Comment:
This is only checked before update, whereas it should be checked before create as well?

Reply:
Isn't checking before update fine?

Reply:
Why? What if I create an order without required fields?